addons/website_payment/static/src/js/payment_form.js

```diff
@@ -62,30 +62,29 @@ PaymentForm.include({
                 'email': _t('Email'),
                 'country_id': _t('Country'),
             };
+            this.el.querySelectorAll("div.invalid-feedback").forEach((el) => el.remove());
```

Comment:
Why not handle these `invalid-feedback` elements directly in the XML using the `required` attribute, as explained in the [Bootstrap documentation](https://getbootstrap.com/docs/5.1/forms/validation/)?